samcli/commands/remote_invoke/invoke/cli.py

```diff
@@ -0,0 +1,139 @@
+"""CLI command for "invoke" command."""
+import logging
+from io import TextIOWrapper
+from typing import cast
+
+import click
+
+from samcli.cli.cli_config_file import TomlProvider, configuration_option
+from samcli.cli.context import Context
+from samcli.cli.main import aws_creds_options, common_options, pass_context, print_cmdline_args
+from samcli.cli.types import RemoteInvokeOutputFormatType
+from samcli.commands._utils.options import remote_invoke_parameter_option
+from samcli.lib.cli_validation.remote_invoke_options_validations import (
+    event_and_event_file_options_validation,
+    stack_name_or_resource_id_atleast_one_option_validation,
+)
+from samcli.lib.remote_invoke.remote_invoke_executors import RemoteInvokeOutputFormat
+from samcli.lib.telemetry.metric import track_command
+from samcli.lib.utils.version_checker import check_newer_version
+
+LOG = logging.getLogger(__name__)
+
+HELP_TEXT = """
+Invoke or send an event to cloud resources in your CFN stack
+"""
+SHORT_HELP = "Invoke a deployed resource in the cloud"
+
+
+@click.command("invoke", help=HELP_TEXT, short_help=SHORT_HELP)
+@configuration_option(provider=TomlProvider(section="parameters"))
+@click.option("--stack-name", required=False, help="Name of the stack to get the resource information from")
+@click.option("--resource-id", required=False, help="Name of the resource that will be invoked")
+@click.option(
```

Comment:
we need similarly named options to have homogenity between local and cloud invoke?

Reply:
Yeah, this is something we will be discussing with the UX team next week. I will raise a follow up PR for the new UX changes to the help text and can add any named option changes there.